evennia/objects/objects.py

```diff
@@ -672,6 +672,55 @@ def for_contents(self, func, exclude=None, **kwargs):
         for obj in contents:
             func(obj, **kwargs)
 
+    def receive_action_msg(self, from_obj, message, mapping, raise_funcparse_errors=False, **kwargs):
```

Comment:
The term `action` doesn't mean much. The point of this is the ability to receive a message that may (optionally) contain actor-stance $You markup. So a better name may be `receive_msg_with_actor_stance`.